examples/wallet/script/src/bin/execute.rs

```diff
@@ -116,3 +115,43 @@ fn main() {
   assert_eq!(charlie_balance, 10);
   assert_eq!(new_alice_balance, alice_balance - 10);
 }
+
+#[cfg(test)]
+mod tests {
+  use athena_interface::{Address, HostDynamicContext, HostStaticContext, MockHost, ADDRESS_ALICE};
+  use athena_sdk::{AthenaStdin, ExecutionClient};
+  use athena_vm_sdk::Pubkey;
+
+  #[test]
+  fn deploying_other_contract() {
```

Comment:
```suggestion
  fn deploy_template() {
```

Let's try to stick to the terminology: "deploy" a "template", and "spawn" a "program" ("instance of a template")